proxygen/lib/http/HTTPHeaders.h

```diff
@@ -238,7 +238,7 @@ class HTTPHeaders {
    * Determines whether header with a given code is a per-hop header,
    * which should be stripped by stripPerHopHeaders().
    */
-  static std::bitset<256>& perHopHeaderCodes();
+  static const std::bitset<256>& perHopHeaderCodes();
```

Comment:
This might be going too far for this pull request. It's a separate commit (currently the latest) so it's easily removed. Just a detail I noticed while making use of the new `HTTPHeaderSet`.